### ISSUE TITLE
feat: make data inspectable in the console

### ISSUE THREAD

**Issue by jimniels** [quadratichq/quadratic#894]
When i'm printing data in the console, it's useful when big ojects have their keys collapsed because then i can inspect/make sense of the data (rather than just dumping it all in the console at once).

This is especially useful when working with APIs and fetching data. Today I have to open the browser devtools and inspect the network call python is making in order to drill into the data and make sense of it.

![CleanShot 2023-11-27 at 20 44 07@2x](https://github.com/quadratichq/quadratic/assets/1316441/c739c376-c317-4f86-ab7b-2cde57f036ac)

Would be nice if this collapse/expand feature worked with data logged to the console.

**Comment by davidkircos** [comment]
I wonder if we should make the console another copy of monaco in readonly mode. It could provide some of this functionality

**Comment by jimniels** [comment]
Yeah that makes sense. It's sort of what I was imagining — we could re-uses it with cold folding etc., but i'm not sure how we'd parse std_out to find and then display the different kinds of data we expect (e.g. structured JSON data, error messages, standard text, etc.)

**Comment by luke-quadratic** [comment]
**Opinion:** this is extremely useful and should be bumped way up in priority; so few data tools of any kind handle this properly and for the ones that do it's just so obvious 

Marking medium prio
